Sources/Stats.php

```diff
@@ -779,7 +779,7 @@ function SMStats()
 		'topics' => $modSettings['totalTopics'],
 		'boards' => 0,
 		'php_version' => $serverVersions['php']['version'],
-		'database_type' => strtolower($serverVersions['db_server']['title']),
+		'database_type' => strtolower($serverVersions['db_engine']['version']),
```

Comment:
The ['db_server']['title'] will only return things like 'mysqli' or 'postgresql'.  Not really useful for it to say 'mysqli' when we need to know 'mysql', 'percona', or 'mariadb'.  ['db_engine']['version'] sends that of those as it is what the engine is reported as from the database itself.  The title is derived from $smcFunc['db_title'] directly and it is something we control in our Subs-Db-*.php directly.  The db_engine comes from $smcFunc['db_get_engine'].  This data is pulled from Subs-Admin.php in the getServerVersions funciton.

In Example on my test server, it would send this data:
  'database_type' => string 'mysqli version' (length=14)
  'database_version' => string '5.6.36-82.0-log' (length=15)

After the change it sends:
  'database_type' => string 'percona' (length=7)
  'database_version' => string '5.6.36-82.0-log' (length=15)

To me that feels like the correct information.